README.md

```diff
@@ -99,6 +99,46 @@ config.settings.DSGVOBanner = {
   privacy_url: '/privacy',
 };
 ```
+You can customize the Style of the Modal Buttons
+```
+config.settings.DSGVOBanner = {
+  bannerAgreeButtonColor: "branded olive",
+  bannerAdjustButtonColor: "branded blue",
+  ...(config.settings.DSGVOBanner || {}),
+}
+```
+Or something like  
+```
+config.settings.DSGVOBanner = {
+  bannerAgreeButtonColor: "red",
+  bannerAdjustButtonColor: "pink",
+  ...(config.settings.DSGVOBanner || {}),
+}
+```
+Keep in mind though, that the 'Adjust privacy Setting' Button will always stay inverted
+
+You can also configure the Banner to not show the first time a User connects to your Site, and only use the Cookie Block and the Disclaimer of the respective Blocks
```

Comment:
In English we don't capitalize every noun like in German.
```suggestion
You can also configure the banner to not show the first time a user connects to your site, and only use the cookie block and the disclaimer of the respective blocks.
```

Reply:
Also, a suggested rewrite to make it a bit more clear:

By default, the banner to configure cookies will be shown in an overlay the first time a user visits the site. You can disable this by setting the `useBanner` setting to `false`. In this case, you must add the "DSGVO Banner" block to a page (such as a privacy settings page) to allow users to configure their cookies.